src/index.html

```diff
@@ -1,13 +1,508 @@
+                      alt="Жінка на лавці"
+                      sizes="(max-width: 767px) 320px, (max-width: 1279px) 400px, (min-width: 1280px) 750px"
+                      class="gallery__image"
+                    >
+                  </a>
+                </li>
+                <li class="gallery__item">
+                  <a href="#image" class="gallery__item-link popup-link">
+                    <img
+                      src="./images/gallery/gallery4-mobile.jpg"
+                      srcset="./images/gallery/gallery4-mobile.jpg 320w, ./images/gallery/gallery4-1x.jpg 400w, ./images/gallery/gallery4-2x.jpg 750w"
+                      alt="Море на сході сонця"
+                      sizes="(max-width: 767px) 320px, (max-width: 1279px) 400px, (min-width: 1280px) 750px"
+                      class="gallery__image"
+                    >
+                  </a>
+                </li>
+              </ul>
+            </div>
+            <ul class="gallery__dots">
+              <li class="gallery__dot active"></li>
+              <li class="gallery__dot"></li>
+              <li class="gallery__dot"></li>
+              <li class="gallery__dot"></li>
+            </ul>
+          </div>
+        </div>
+      </section>
+
+      <section class="section main__describe" id="describe">
+        <div class="section__container">
+          <div class="describe js-anim-items">
+            <div class="describe__content">
+              <h2 class="describe__title anim-show">Підписка</h2>
+              <p class="describe__description anim-show">Бути в курсі всіх актуальних подій та останніх новин.</p>
+              <form
+                action="mailto:hello@miami.com"
+                name="describe-form"
+                enctype="text/plain"
+                method="post"
+                class="form describe__form"
+              >
+                <input
+                  type="email"
+                  name="email"
+                  placeholder="E-mail"
+                  required
+                  class="form__fields"
+                  autocomplete="on"
+                >
+                <button type="submit" class="form__button">
+                  <svg
+                    class="icon icon__button form__icon-button"
+                    width="24"
+                    height="24"
+                    aria-label="Підписка"
+                  >
+                    <use href="./images/icons/sprite.svg#arrow-button"></use>
+                  </svg>
+                </button>
+              </form>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer class="footer js-anim-items">
+      <div class="footer__container">
+        <div class="footer__content">
+          <div class="footer__contacts" id="contacts">
+            <div class="footer__logo-container">
+              <div class="logo footer__logo">
+                <a href="#home">na<br>mu</a>
+              </div>
```

Comment:
In this case this logic is a good one because I will scroll to top from the bottom